CHANGELOG.md

```diff
@@ -8,17 +8,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### External
```

Comment:
@mplorentz @bryanmontz @martindsq thoughts on this change to the CHANGELOG? Before I explain what it is, do these new headings make sense to you?

Reply:
At first I thought they were external vs internal contributors. But after looking more closely I think maybe external is things that users will notice and internal is not? So I guess I found the names a little confusing. Maybe "UX changes" and "Dev UX" changes or something like that would be more clear?

We have to do this kind of sorting before publishing the release notes anyway, so I guess it wouldn't hurt to do it here. Although I think it would be best to update the `stamp_release` fastlane lane to auto-generate these headers for consistency.

Reply:
Yeah, so the idea here is that "External" means things that we'll publish in our release notes on TestFlight. Since we don't want our users to see things like "Replaced hard-coded color values.", those notes would go under "Internal". I wonder if "Public" and "Internal" would work?

I'm all for updating `stamp_release` if/when we decide on a new approach, and I'd be happy to do that.

Reply:
Oh, I guess we could instead have a section called "Release Notes" and one called... I dunno, I keep coming up with "Internal". "Non Release Notes"? "Dev Notes"? "For the devs"?

Reply:
I'm reverting the changes in this PR and will create a new PR with just CHANGELOG format changes. I'd still love to hear thoughts here or in the future PR.